code/game/objects/items/stacks/stack.dm

```diff
@@ -339,7 +339,7 @@
 	return ..()
 
 /obj/item/stack/restock()
-	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical))
+	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical) || istype(src,/obj/item/stack/package_wrap))
```

Comment:
I would suggest changing that += 2 right there to a variable like 'restock amount' that defaults to zero and you can set it to whatever value you want on those particular types. And add an ```if(!restock_amount) return```.